sections/introduction.tex

```diff
@@ -4,57 +4,46 @@ \section{Introduction}\label{sec:introduction}
+Within the field of topic modeling, \Gls{lda} is a well known topic model which generates topics and topic distributions for documents based on the words in documents~\cite{blei2003latent}, without the use of metadata.
+Other extensions of \gls{lda} have also modeled various other information to generate better topics and/or topics with different focuses and potential uses.
+In most models metadata is not used.
+However, there is some previous work that attempts to make use of the metadata available, by incorporating it into topic models, such as the Author-Topic model by \citet{author_topic_2012} and the MetaLDA model by \citet{MetaLDA2017}.
 
 % Author-Topic
 One of the well-known extensions of \gls{lda} is the Author-Topic model by \citet{author_topic_2012} that combines the \gls{lda} model with author information to model the relationship between authors and the documents, which they have written.
 This is based on the assumption that most authors usually write about only a few different topics.
 By modeling this relationship, they find connections between authors and topics, and between different authors, giving more underlying information to the topics.
 They also show that the Author-Topic combination yields better and more coherent topics, which begs the question of whether any other document-related data can be applied similarly.
-\vejleder[inline]{what cant they do? what are the weaknesses? this will sharpen the part about our work.}
+However, they only test their algorithm on scientific papers, where usually the authors write about the same subject(their research field), which might not be the case for other fields, like journalism. 
 
 % In this paper
-In this paper, we will be taking a closer look at a specific dataset and construct different topic models that incorporate various metadata.
-We do this in order to see if they can improve the quality of the topics or the efficiency of the model.
-We will also investigate the differences between the produced topics for various models, and evaluate their potential uses.
-
-\vejleder[inline]{skriv introduktionen frem til dette punkt mere sammen}
+In this paper, we are investigating use cases for the dataset, that has been provided by Nordjyske, within the field of topic modelling. 
+We want to construct different topic models that incorporates various metadata, to see whether incorporating this information, changes the topic quality.
+We do this to aid in solving, potential problems that Nordjyske might encounter as a media group, such as recommendation, search.
+We also investigate differences between the produced topics for various models, to evaluate their potential uses.
 
 % Problem Statement
 We define the following problem to investigate in this paper:
 
 \begin{itemize}
 	\item \textit{How does including metadata within the \gls{lda} model impact the resulting topics?}
```

Comment:
```suggestion
	\item \textit{How does including metadata within the \gls{lda} model impact the resulting topics?}
	\item \textit{What observations can be made in models trained with metadata?}
```
Hvis vi vil ligge det frem som et område vi undersøger, for at understøtte en dybere analyse/diskussion.

Reply:
Jeg ved ikke lige hvad jeg syntes om den ekstra problem formulering

Reply:
Det er også bare et forslag til hvis vi ville lede mere ind til analysen. Man kan muligvis godt sige at "How does including metadata within the \gls{lda} model impact the resulting topics?" allerede dækker analyse af modellerne.

Reply:
ja tror at den nuværender også dækker